app/cli/internal/action/attestation_status.go

```diff
@@ -170,6 +199,37 @@ func (action *AttestationStatus) Run(ctx context.Context, attestationID string)
 	return res, nil
 }
 
+// getPolicyEvaluations retrieves both material-level and attestation-level policy evaluations
+func (action *AttestationStatus) getPolicyEvaluations(ctx context.Context, c *crafter.Crafter, statement *intoto.Statement) (map[string][]*PolicyEvaluation, error) {
+	// grouped by material name
+	evaluations := make(map[string][]*PolicyEvaluation)
+
+	// Add material-level policy evaluations
+	for _, v := range c.CraftingState.Attestation.GetPolicyEvaluations() {
+		if existing, ok := evaluations[v.MaterialName]; ok {
+			evaluations[v.MaterialName] = append(existing, policyEvaluationStateToActionForStatus(v))
+		} else {
+			evaluations[v.MaterialName] = []*PolicyEvaluation{policyEvaluationStateToActionForStatus(v)}
+		}
+	}
+
+	// Add attestation-level policy evaluations
+	attestationEvaluations, err := c.EvaluateAttestationPolicies(ctx, statement)
```

Comment:
If `statement` comes from `renderer.Statement()`, then this call is not needed, because policies are already evaluated in that method.

Reply:
I think you might be talking about `Render` which is the final output as DSSE envelope. `renderer.statement` is a new method that doesn't have the policies yes, afaik

Reply:
you were right, see the new change, thanks